src/scss/components/popover/_index.scss

```diff
@@ -1,8 +1,10 @@
 @import "vars";
 @import "mixins";
 @import "../config";
+@import "../../base";
+@import "../../elements/_has-arrow";
```

Comment:
I'm not sure if this imports should be here, maybe there is a better way of exposing this globally?

Reply:
Going this route, we would have a different implementation from the current HelixUI components.